docker/php/docker-entrypoint.sh

```diff
@@ -6,12 +6,12 @@ if [ "${1#-}" != "$1" ]; then
 	set -- php-fpm "$@"
 fi
 
-if [ "$1" = 'php-fpm' ] || [ "$1" = 'bin/console' ]; then
-    PHP_INI_RECOMMENDED="$PHP_INI_DIR/php.ini-production"
-    if [ "$APP_ENV" != 'prod' ]; then
-        PHP_INI_RECOMMENDED="$PHP_INI_DIR/php.ini-development"
-    fi
-    ln -sf "$PHP_INI_RECOMMENDED" "$PHP_INI_DIR/php.ini"
+if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
+	PHP_INI_RECOMMENDED="$PHP_INI_DIR/php.ini-production"
+	if [ "$APP_ENV" != 'prod' ]; then
+		PHP_INI_RECOMMENDED="$PHP_INI_DIR/php.ini-development"
+	fi
+	ln -sf "$PHP_INI_RECOMMENDED" "$PHP_INI_DIR/php.ini"
 
     # The first time volumes are mounted, the project needs to be recreated
```

Comment:
What is the reason for this? Let's see if we could figure this one out.

Reply:
Because the files of the project itself aren't present on the host, and you cannot (AFAIK) create a volume sharing file from the inside of the container to the host, you have to do it the other way around.

Reply:
Maybe could we create an internal volume with `VOLUME` and then mount it with Docker Compose? I'm not sure if it could work, but it could be tried.

Reply:
Actually, we should not run `composer create-project` inside Dockerfile. It makes no sense. If it's to make sure the project is in sync with `symfony/skeleton`, it should be done externally.

Reply:
But doing it "externally" is by nature incompatible with the main objective of this project: you must be able to bootstrap a new Symfony project by just running `docker-compose up`, without having PHP, Composer, Bash or anything else installed on the host computer.
The goal is to provide a similar experience than with API Platform, or Laravel Homestead.

If we assume that the user has PHP and Composer installed, we could just ship the relevant Dockerfile and `docker-compose.yaml` files in the recipes.

Reply:
Sorry for the confusion. I meant the syncing must be done externally (maybe by a bot operated by the Symfony team). The project must indeed be self-contained. Currently that's not the case, because you're creating the project when building the Docker image.

Reply:
The current approach is definitely a hack.
Creating a bot that regenerating this skeleton every time recipe change (and when a new Symfony version is released) would be definitely cleaner (it was my 1st idea), however when I created this repository, I haven't managed a good way to detect such events in an easy way.
While thinking about it, I'm sure @fabpot already has some tools like that up and running for Flex and the existing skeletons. Fabien, do you think we could trigger such bot on an existing hook?